VirtoCommerce.CustomerModule.Web/Scripts/blades/filter-detail.tpl.html

```diff
@@ -0,0 +1,77 @@
+<div class="blade-content">
+    <div class="blade-inner">
+        <div class="inner-block">
+            <form name="formScope">
+                <div class="form-group" ng-init="setForm(formScope)">
+                    <label class="form-label">{{ 'customer.blades.filter-detail.labels.name' | translate }}</label>
+                    <div class="form-input">
+                        <input ng-model="blade.currentEntity.name" required placeholder="{{ 'customer.blades.filter-detail.placeholders.name' | translate }}" />
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="form-label">{{ 'customer.blades.filter-detail.labels.criteria' | translate }}</label>
+                    <div class="form-input">
+                        <input ng-model="blade.currentEntity.keyword" placeholder="{{ 'customer.blades.filter-detail.placeholders.criteria' | translate }}" />
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="form-label">{{ 'customer.blades.filter-detail.labels.searchInVariations' | translate }}</label>
+                    <div class="form-input">
+                        <label class="form-label __switch">
+                            <input type="checkbox" ng-model="blade.currentEntity.searchInVariations">
+                            <span class="switch"></span>
+                        </label>
+                    </div>
+                </div>
+            </form>
+            <div class="form-group">
+                <div class="list __info">
+                    <div class="list-t">org1.com</div>
+                    <div class="list-descr">all contacts with word 'org1.com' in property values</div>
```

Comment:
Could you move these descriptions to localizations?

Reply:
Maybe need to tell what is 'property values' - e.g 'in searchable indexed property values (values in index '__content' array)'

Reply:
Moved